src/externals/bridge/bridge.module.ts

```diff
@@ -0,0 +1,19 @@
+import { Module } from '@nestjs/common';
+import { ConfigModule } from '@nestjs/config';
+import { CryptoService } from '../crypto/crypto';
+import { HttpClientModule } from '../http/http.module';
+import { BridgeService } from './bridge.service';
+
+@Module({
+  imports: [ConfigModule, HttpClientModule],
+  controllers: [],
+  providers: [
+    BridgeService,
+    {
+      provide: CryptoService,
+      useValue: new CryptoService(),
```

Comment:
Why do you have to do this like this with CryptoService here?

Reply:
The `CryptoService` was not made "@Injectable()", instead is a singleton. I could use `CryptoService.getInstance()` too